### ISSUE TITLE
GPL-613 Start docker services in order

### ISSUE THREAD

**Issue by emrojo** [sanger/lighthouse#81]
**User story**
This is a story for psd-deployment project.
When restarting the stack the services take a while to connect with each other. We have to check if this is because they are not started in order. If that is the case, we may need to add dependencies in the config for the stack so they can start in a better order

**Who are the primary contacts for this story**
Eduardo Martin

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Services start in order without sending errors about not connecting to dependent services.

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by KatyTaylor** [comment]
Closing in favour of larger refactor story - https://github.com/sanger/deployment/issues/5

Also we don't think it's good practice to specify load order, it should handle it in other ways
